tests/oss-fuzz/build.sh

```diff
@@ -43,14 +43,13 @@ export CFLAGS=""
 export CXXFLAGS=""
 
 # fuzz flags are problematic with meson (hence no dav1d) and no point in fuzzing fuzztest.
```

Comment:
The last part of the sentence needs an update.

Reply:
Done